### PR TITLE
Misc updates to docs

### PR DESCRIPTION
**Pull request by maietta**
Updated docs to include that Caprover users like myself and at least one other have tested this Linux distribution as being good. I have performed extensive testing of both single node and multi-node swarm using VPS accounts and also on a single node swarm on a Raspberry Pi 4 8GB with no issues detected.

### CODE REVIEW

File: website/i18n/en.json [hunk 1]
```diff
@@ -45,6 +49,10 @@
         "title": "Database Connection",
         "sidebar_label": "Database Connection"
       },
+      "deploy-from-github": {
```
Comment (githubsaturn) [inline website/i18n/en.json:52]: This should be a subpage of CI/CD.

File: docs/get-started.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ _**CPU Architecture**:_ CapRover source code is compatible with any CPU architec
 
 _**Recommended Stack**:_ CapRover is tested on Ubuntu 18.04 and Docker 19.03. If you're using CapRover on a different OS, you might want to look at [Docker Docs](https://docs.docker.com/engine/userguide/storagedriver/selectadriver/#supported-storage-drivers-per-linux-distribution).
 
+_**Ubuntu 22.04**:_ The amd64 and aarch64 distributions hav been tested by members of the Caprover community to be fully functional since it's release. Ubuntu 22.04 will continue to receive security updates until April 2027 at least.
```
Comment (githubsaturn) [inline docs/get-started.md:45]: nit:
it's => its

File: docs/support.md [hunk 3]
```diff
@@ -29,6 +29,9 @@ Alternatively, you can use the following third party entities for their paid ser
 
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; We have been using Caprover in many project including government server as well as private servers with single and miltiple instances for large scaled applications (with more than 500k traffic) to small scaled. Mainly we used caprover for hosting dockerized PHP applicationa and other reactjs and vuejs based static web applications. If you need any help, you can contact us: [yourfriends@integrid.solutions](mailto:yourfriends@integrid.solutions)
 
+##### [PremoWeb Internet Services](https://premoweb.com)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Got a complicated application stack or many applications you'd like to move to Caprover? PremoWeb is a United States based business now offering application migration services for those who want to make the migration. Schedule a free consulation with owner Nick Maietta at 888-PREMOWEB (888-773-6693) or via email at `nick at/premoweb/dot/com`. You can also find Nick on the [CapRover Slack Group](https://join.slack.com/t/caprover/shared_invite/zt-o8obmx7o-x9icEG5fDAtvE9zhHlltzQ).
+
```
Comment (githubsaturn) [inline docs/support.md:34]: Nice! 

You can remove the link, it's already in the footer of the site. Duplicating links make it more error prone when we need to update the invite link (it happens every now and then when the previous link gets abused by bots).

**Review by githubsaturn — COMMENTED** [review]
Perfect, thanks!

File: website/i18n/en.json (hunk 1 above)
Comment (maietta) [reply website/i18n/en.json:52]: This is my first time using Docusourous. I have not quite figured out how to get the nested navigation working despite reading over the official docs. I may need your assistance on this.

File: docs/get-started.md (hunk 2 above)
Comment (maietta) [reply docs/get-started.md]: Corrected in last commit.

File: docs/support.md (hunk 3 above)
Comment (maietta) [reply docs/support.md:34]: I've removed the link in the last commit, but don't see that commit here. Marking is resolved for the moment.

File: website/i18n/en.json (hunk 1 above)
Comment (maietta) [reply website/i18n/en.json]: Okay, figured it out. Version 1.x of Docusourus is being used instead of 2.x.

Comment (maietta) [reply website/i18n/en.json]: I've moved renamed, re-titled and moved the existing CI/CD Integration page as being a nested page under CI/CD Integration subcategory page and now referring to it as Deploy from GitLab as this page almost completely is devoted to GitLab, with the exception of the "Alternative Method" at the bottom of the page. This visually looks better to me in the navigation as we now have both Deploy from GitHub as well as a Deploy from GitLab page. I'll be working on an automation tool for both GitLab and Bitbucket in the very near future, to which we can then update the existing GitLab page for simplicity for users and add support for Bitbucket. For now, hope this is more in line with what you invision. Let me know.

**Comment by githubsaturn** [comment]
LGTM! Thanks!

**Pull request merged**